wrap/Module.cpp

```diff
@@ -350,7 +350,10 @@ void Module::emit_cython_pxd(FileWriter& pxdFile) const {
                  "        T* get()\n"
                  "        long use_count() const\n"
                  "        T& operator*()\n\n"
-                 "    cdef shared_ptr[T] dynamic_pointer_cast[T,U](const shared_ptr[U]& r)\n"
+                 "    cdef shared_ptr[T] dynamic_pointer_cast[T,U](const shared_ptr[U]& r)\n\n";
+
+  // gtsam alignment-friendly shared_ptr
```

Comment:
how does this guarantee the correct make_shared is used everywhere?

Reply:
If you take a look at the generated `gtsam.cpp` you will see that `boost::make_shared` is replace by this everywhere.